tests/manage/pv_services/test_daemon_kill_during_pvc_pod_deletion_and_io.py

```diff
@@ -1,132 +1,164 @@
 import logging
 from concurrent.futures import ThreadPoolExecutor
 from itertools import cycle
+from time import sleep
 
 import pytest
 from functools import partial
 
-from ocs_ci.framework.testlib import ManageTest, tier4, tier4b
+from ocs_ci.framework.testlib import ManageTest, tier4, tier4c, polarion_id
 from ocs_ci.framework import config
 from ocs_ci.ocs import constants, node
-from ocs_ci.ocs.resources.pvc import get_all_pvcs, delete_pvcs
+from ocs_ci.ocs.resources.pvc import delete_pvcs
 from ocs_ci.ocs.resources.pod import (
     get_mds_pods,
     get_mon_pods,
     get_mgr_pods,
     get_osd_pods,
-    get_all_pods,
     get_fio_rw_iops,
-    get_plugin_pods,
-    get_rbdfsplugin_provisioner_pods,
-    get_cephfsplugin_provisioner_pods,
-    get_operator_pods,
 )
-from ocs_ci.utility.utils import TimeoutSampler, ceph_health_check
+from ocs_ci.utility.utils import TimeoutSampler, ceph_health_check, run_cmd
 from ocs_ci.helpers.helpers import (
     verify_volume_deleted_in_backend,
     wait_for_resource_state,
-    wait_for_resource_count_change,
     verify_pv_mounted_on_node,
     default_ceph_block_pool,
     select_unique_pvcs,
 )
-from ocs_ci.helpers import disruption_helpers
+from ocs_ci.helpers import disruption_helpers, helpers
 
 log = logging.getLogger(__name__)
 
 
 @tier4
-@tier4b
-@pytest.mark.parametrize(
-    argnames=["interface", "resource_name"],
-    argvalues=[
-        pytest.param(
-            *[constants.CEPHBLOCKPOOL, "mgr"], marks=pytest.mark.polarion_id("OCS-1136")
-        ),
-        pytest.param(
-            *[constants.CEPHBLOCKPOOL, "mon"], marks=pytest.mark.polarion_id("OCS-1122")
-        ),
-        pytest.param(
-            *[constants.CEPHBLOCKPOOL, "osd"], marks=pytest.mark.polarion_id("OCS-1129")
-        ),
-        pytest.param(
-            *[constants.CEPHFILESYSTEM, "mgr"],
-            marks=pytest.mark.polarion_id("OCS-1108"),
-        ),
-        pytest.param(
-            *[constants.CEPHFILESYSTEM, "mon"],
-            marks=pytest.mark.polarion_id("OCS-1093"),
-        ),
-        pytest.param(
-            *[constants.CEPHFILESYSTEM, "osd"],
-            marks=pytest.mark.polarion_id("OCS-1101"),
-        ),
-        pytest.param(
-            *[constants.CEPHFILESYSTEM, "mds"],
-            marks=pytest.mark.polarion_id("OCS-1115"),
-        ),
-    ],
-)
-class TestDaemonKillDuringMultipleDeleteOperations(ManageTest):
+@tier4c
+class TestDaemonKillDuringMultipleCreateDeleteOperations(ManageTest):
```

Comment:
Don't we want to modify the file name too?

Reply:
Yes, I did that in one commit and reverted for the easiness to review the changes by comparison. The file name to be modified immediately after merging this PR.